bundles/binding/org.openhab.binding.dsmr/README.md

```diff
@@ -1,139 +1,158 @@
+| gValueX            | Meter reading *X* of the buffer (hourly period)|v3.0<sup>2</sup><br>*X* is a value [2-10]<br>Higher values are older|m3
+| gProfileStatus     | Unknown|v3.0|
+| gRecordingPeriod   | Duration of a value recording|v3.0|minutes
+| gValueCompensated  | Temperature compensated delivery of the last 24 hours|v2.1 and v2.2|m3
+| gValvePosition     | Valve position|All versions|
+| **Heating meter values**||||
+| hDeviceType        | Device Type|v3.0 and up|
+| hEquipmentId       | Equipment identifier|All versions|
+| hValueTS           | Timestamp of the last measurement (local time)|v4.0 and up|Date &amp; time
+| hValue             | Last hour delivery|All versions|GJ
+| hValvePosition     | Valve position|v3.0 and up|
+| **Cooling meter values**||||
+| cDeviceType        | Device Type|v3.0 and up|
+| cEquipmentId       | Equipment identifier|All versions|
+| cValueTS           | Timestamp of the last measurement (local time)|v4.0 and up|Date &amp; time
+| cValue             | Last hour delivery|All versions|GJ
+| cValvePosition     | Valve position|v3.0 and up|
+| **Cooling meter values**||||
+| wDeviceType        | Device Type|v3.0 and up|
+| wEquipmentId       | Equipment identifier|All versions|
+| wValueTS           | Timestamp of the last measurement (local time)|v4.0 and up|Date &amp; time
+| wValue             | Last hour delivery|All versions|m3
+| wValvePosition     | Valve position|v3.0 and up|
+| **Generic meter values**||||
+| genericDeviceType  | Device Type|v3.0|
+| genericEquipmentId | Equipment identifier|v3.0|
+| genericValue       | Last hour delivery|v3.0|
+| genericValvePosition |Valve/Switch position|v3.0|
+| **Slave electricity meter values**||||
+| seDeviceType       | Device Type|v4.0 and up|
+| seEquipmentId      | Equipment identifier|v4.0 and up|
+| seValueTS          | Timestamp of the last measurement (local time)|v4.0 and up|Date &amp; time
+| seValue            | Last hour delivery|v4.0 and up|kWh
+| seSwitchPosition   | Switch position|v4.0 and up
+
+<sup>1</sup> This item isn't part of any specification, but the ITRON ACE4000 GTMM Mk3 does use this value.<br/>
 <sup>2</sup> Gas values for DSMR v3.0 are available in a list of max. 10 entries. The binding assumes the first value is the most recent (and thus available in `gValue`)
-<h3>Examples</h3>
+
+## Examples
+
+### Item configuration examples
+
 ```
 Number P1_Actual_Delivery "Actual usage [%.3f kW]" {dsmr="eActualDelivery"}
 Number P1_Meter_DeliveryLow "Meterstand reading low tariff [%.3f kWh]" {dsmr="eDeliveryTariff1"}
 Number P1_Meter_DeliveryNormal "Meter reading normal tariff[%.3f kWh]" {dsmr="eDeliveryTariff2"}
 ```
-<h2>Test results</h2>
+
+
+## Notes
+
+### Logging
+Use the following setting in logback.xml to increase verbosity of the logging
+```
+<logger name="org.openhab.binding.dsmr" level="DEBUG"/>
+```
+
```

Comment:
logback.xml is gone in openHAB 2.  I've taken to just reporting the name of the logger (`org.openhab.binding.dsmr` in this case) and saying it can be set to DEBUG for diagnosis and back to INFO or DEFAULT afterwards (in whatever language you think is good).  Also, Jekyll needs blank lines around headers and code fences.